app/gzac/src/main/java/com/ritense/gzac/listener/ApplicationReadyEventListener.kt

```diff
@@ -111,12 +114,54 @@ class ApplicationReadyEventListener(
             createBomenObjectManagement(objecttypenApiPluginId, objectenApiPluginId)
             createVerzoekPlugin(notificatiesApiPluginId, bezwaarConfigurationId)
             createSmartDocumentsPlugin()
-            createPortaalTaakPlugin(notificatiesApiPluginId, taakConfigurationId)
+            val protaalTaakPluginId = createPortaalTaakPlugin(notificatiesApiPluginId, taakConfigurationId)
+            createPortalPersonToPortaalTaakLink(protaalTaakPluginId)
         } catch (ex: Exception) {
             throw RuntimeException("Failed to deploy plugin configurations for development", ex)
         }
     }
 
+    private fun createPortalPersonToPortaalTaakLink(protaalTaakPluginId: UUID) {
+        val portalPersonProcessDefinitionId = repositoryService.createProcessDefinitionQuery()
+            .processDefinitionKey("portal-person")
+            .latestVersion()
+            .singleResult()
+            .id
+        if (pluginService.getProcessLinks(portalPersonProcessDefinitionId, "portal-task").isEmpty()) {
+            pluginService.createProcessLink(
+                PluginProcessLinkCreateDto(
+                    processDefinitionId = portalPersonProcessDefinitionId,
+                    activityId = "portal-task",
+                    activityType = "bpmn:UserTask:create",
+                    pluginConfigurationId = protaalTaakPluginId,
+                    pluginActionDefinitionKey = "create-portaaltaak",
+                    actionProperties = jacksonObjectMapper().readValue(
```

Comment:
Looks like test data

Reply:
It is data for testing the portaaltaak-plugin locally when starting the application. This is what the class `ApplicationReadyEventListener.kt` is used for